src/settings.rs

```diff
@@ -50,6 +50,10 @@ pub struct ServiceConfig {
 
     /// How much time is considered too long for a query to execute.
     pub slow_query_secs: Option<f32>,
+
+    /// Whether to enable reporting of measured hardware utilization in API responses.
+    #[serde(default)]
+    pub hardware_reporting: bool,
```

Comment:
Let's make it optional, usually it helps for maintaining param compatibility in future